src/quartic/pipeline/validator/git.py

```diff
@@ -0,0 +1,51 @@
+import uuid
+import os
+from git import Repo, RemoteProgress
+from quartic.common import yaml_utils, utils
+import click
+
+class MyProgressPrinter(RemoteProgress):
+    def __init__(self):
+        RemoteProgress.__init__(self)
+        self.pbar = click.progressbar(length=3)
+    def update(self, op_code, cur_count, max_count=None, message=""):
+        self.pbar.update(1)
+
+def check_git_dir():
+    """Expect config to be in the same place as root of git repo."""
+    cpath = yaml_utils.config_path()
+    cdir = os.path.dirname(cpath)
+    return os.path.isdir(os.path.join(cdir, ".git"))
```

Comment:
Does git python have any functions for checking if this is a git repo?

Reply:
I don't think so but can check.

Reply:
Don't think so. Will check. What's wrong with this though?

Reply:
Nothing. Only better to use existing tested code if it's there rather than rolling our own (given we've imported it already). Fine if not though.